sample/src/main/res/layout/fragment_colors.xml

```diff
@@ -3,18 +3,17 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:paddingStart="16dp"
-    android:paddingEnd="16dp">
+    android:layout_height="match_parent">
 
     <androidx.appcompat.widget.LinearLayoutCompat
+        android:padding="16dp"
```

Comment:
Why didn't you put `@dimen/spacing_medium` here ?

Reply:
Mistake.. I forgot to do the work for the colors and typography screens. This is now fixed